src/controller/CommentController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * CommentController File Doc Comment
+ * 
+ * PHP Version 8.1.10
+ * 
+ * @category Controller
+ * @package  App\controller
+ * @author   Marine Sanson <marine_sanson@yahoo.fr>
+ * @license  https://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     https://www.blog.marinesanson.fr/ Not inline for the moment
+ */
+declare(strict_types=1);
+
+namespace App\controller;
+
+use App\controller\AbstractController;
+use App\repository\CommentRepository;
+
+/**
+ * CommentController Class Doc Comment
+ * 
+ * @category Controller
+ * @package  App\controller
+ * @author   Marine Sanson <marine_sanson@yahoo.fr>
+ * @license  https://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     https://www.blog.marinesanson.fr/ Not inline for the moment
+ */
+class CommentController extends AbstractController
+{
+
+    /**
+     * Summary of _commentRepository
+     * 
+     * @var CommentRepository
+     */
+    private $_commentRepository;
```

Comment:
c'est un service qui doit etre dans le controller pas un repository
`controler - service - repository `
```suggestion
    private CommentService $_commentService;
```

Reply:
ok fait